packages/twitch/src/API/Helix/Stream/HelixStream.ts

```diff
@@ -149,6 +149,16 @@ export class HelixStream {
 		return this._data.thumbnail_url;
 	}
 
+	/**
+	 * The URL of the thumbnail of the stream with a width and height.
```

Comment:
Rather than describing the *result*, methods should be documented as an *action* (i.e. `Builds the URL ... using the given dimensions`)

Reply:
Done